upgrade_logic/business_objects/gateway.py

```diff
@@ -11,6 +11,26 @@
 from submodules.model import enums
 
 
+def gateway_1_15_0() -> bool:
+    # here, we update data for cognition using the gateway pattern
+    # as the corresponding database updates (alembic) are managed using the refinery gateway it is
+    # ensured that these updates are executed at the correct time
+    __gateway_1_15_0_add_cognition_project_file_defaults()
+    return True
+
+
+def __gateway_1_15_0_add_cognition_project_file_defaults() -> bool:
+    query = """
+    UPDATE cognition.project
+    SET max_file_size_mb = 3,
+        allow_file_upload = FALSE
+    WHERE max_file_size_mb IS NULL
+    """
+    general.execute(query)
+    general.commit()
+    return True
```

Comment:
I wonder if it makes sense to also update the fields that have been added to conversation. From a quick glance over the code I think we don't run into any errors, but we would have a more uniform status

Reply:
initially decided against it since i didn't want to create something with updater then i needed it and didn't add for converstaion :D => will now